AX: check-webkit-style: DRT/WKTR - resolve all remaining warnings

dusek opened the pull request:
#### fee6ad48799e1767fe1711ce6eb7c65496d3e29b
<pre>
AX: check-webkit-style: DRT/WKTR - resolve all remaining warnings
<a href="https://bugs.webkit.org/show_bug.cgi?id=297334">https://bugs.webkit.org/show_bug.cgi?id=297334</a>

Reviewed by Tyler Wilcock.

Resolves last 25 warnings previously reported by check-webkit-style in DRT/WKTR accessibility code:
$ Tools/Scripts/check-webkit-style Tools/DumpRenderTree/{,*}/Accessibility* Tools/WebKitTestRunner/InjectedBundle/{,*}/Accessibility*
Total errors found: 0 in 48 files

* Tools/DumpRenderTree/AccessibilityController.cpp:
* Tools/DumpRenderTree/AccessibilityTextMarker.h:
* Tools/DumpRenderTree/AccessibilityUIElement.cpp:
* Tools/DumpRenderTree/mac/AccessibilityControllerMac.mm:
* Tools/DumpRenderTree/mac/AccessibilityNotificationHandler.mm:
* Tools/DumpRenderTree/mac/AccessibilityTextMarkerMac.mm:
* Tools/DumpRenderTree/mac/AccessibilityUIElementMac.mm:
(selectTextParameterizedAttributeForCriteria):
(AccessibilityUIElement::parameterizedAttributeNames):
(AccessibilityUIElement::boundsForRange):
(AccessibilityUIElement::columnIndexRange):
* Tools/WebKitTestRunner/InjectedBundle/AccessibilityTextMarker.cpp:
* Tools/WebKitTestRunner/InjectedBundle/AccessibilityTextMarkerRange.cpp:
* Tools/WebKitTestRunner/InjectedBundle/AccessibilityUIElement.cpp:
* Tools/WebKitTestRunner/InjectedBundle/atspi/AccessibilityUIElementAtspi.cpp:
(WTR::AccessibilityUIElement::stringAttributeValue):
(WTR::AccessibilityUIElement::sortDirection const):
* Tools/WebKitTestRunner/InjectedBundle/cocoa/AccessibilityCommonCocoa.mm:
* Tools/WebKitTestRunner/InjectedBundle/ios/AccessibilityControllerIOS.mm:
* Tools/WebKitTestRunner/InjectedBundle/ios/AccessibilityTextMarkerIOS.mm:
* Tools/WebKitTestRunner/InjectedBundle/mac/AccessibilityNotificationHandler.mm:
* Tools/WebKitTestRunner/InjectedBundle/mac/AccessibilityUIElementMac.mm:
(WTR::AccessibilityUIElement::boundsForRange):
* Tools/WebKitTestRunner/InjectedBundle/win/AccessibilityUIElementWin.cpp:

Canonical link: <a href="https://commits.webkit.org/298630@main">https://commits.webkit.org/298630@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/84c507f3c52b8efe9fa9cbd9e7233cb62dbbc01b

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/116110 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/131/builds/35771 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/138/builds/26315 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/122166 "Built successfully") | [✅ 🛠 win](https://ews-build.webkit.org/#/builders/59/builds/66665 "Built successfully") 
| [✅ 🧪 bindings](https://ews-build.webkit.org/#/builders/9/builds/117999 "Passed tests") | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/130/builds/36465 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/123/builds/44359 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/88213 "Passed tests") | [  ~~🧪 win-tests~~](https://ews-build.webkit.org/#/builders/60/builds/42752 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/119058 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/132/builds/29093 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/104187 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/68624 "Passed tests") | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/133/builds/28211 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/135/builds/22294 "Passed tests") | [✅ 🛠 wpe-cairo](https://ews-build.webkit.org/#/builders/65/builds/65848 "Built successfully") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/98484 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/136/builds/22431 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/125316 "Built successfully") | 
| | [✅ 🛠 vision](https://ews-build.webkit.org/#/builders/128/builds/43004 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/122/builds/32286 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/96946 "Passed tests") | 
| | [✅ 🛠 vision-sim](https://ews-build.webkit.org/#/builders/121/builds/43369 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/100376 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/96730 "Passed tests") | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/24614 "Built successfully and passed tests") | [✅ 🧪 vision-wk2](https://ews-build.webkit.org/#/builders/126/builds/42002 "Passed tests") | [✅ 🧪 mac-intel-wk2](https://ews-build.webkit.org/#/builders/137/builds/19890 "Passed tests") | [✅ 🛠 playstation](https://ews-build.webkit.org/#/builders/134/builds/38950 "Built successfully") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/127/builds/42891 "Built successfully") | | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/125/builds/42358 "Built successfully") | | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/129/builds/45693 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/124/builds/44062 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

webkit-early-warning-system commented:
Starting EWS tests for https://github.com/WebKit/WebKit/commit/84c507f3c52b8efe9fa9cbd9e7233cb62dbbc01b. Live statuses available at the PR page, https://github.com/WebKit/WebKit/pull/49336

dusek committed:
AX: check-webkit-style: DRT/WKTR - resolve all remaining warnings
https://bugs.webkit.org/show_bug.cgi?id=297334

Reviewed by Tyler Wilcock.

Resolves last 25 warnings previously reported by check-webkit-style in DRT/WKTR accessibility code:
$ Tools/Scripts/check-webkit-style Tools/DumpRenderTree/{,*}/Accessibility* Tools/WebKitTestRunner/InjectedBundle/{,*}/Accessibility*
Total errors found: 0 in 48 files

* Tools/DumpRenderTree/AccessibilityController.cpp:
* Tools/DumpRenderTree/AccessibilityTextMarker.h:
* Tools/DumpRenderTree/AccessibilityUIElement.cpp:
* Tools/DumpRenderTree/mac/AccessibilityControllerMac.mm:
* Tools/DumpRenderTree/mac/AccessibilityNotificationHandler.mm:
* Tools/DumpRenderTree/mac/AccessibilityTextMarkerMac.mm:
* Tools/DumpRenderTree/mac/AccessibilityUIElementMac.mm:
(selectTextParameterizedAttributeForCriteria):
(AccessibilityUIElement::parameterizedAttributeNames):
(AccessibilityUIElement::boundsForRange):
(AccessibilityUIElement::columnIndexRange):
* Tools/WebKitTestRunner/InjectedBundle/AccessibilityTextMarker.cpp:
* Tools/WebKitTestRunner/InjectedBundle/AccessibilityTextMarkerRange.cpp:
* Tools/WebKitTestRunner/InjectedBundle/AccessibilityUIElement.cpp:
* Tools/WebKitTestRunner/InjectedBundle/atspi/AccessibilityUIElementAtspi.cpp:
(WTR::AccessibilityUIElement::stringAttributeValue):
(WTR::AccessibilityUIElement::sortDirection const):
* Tools/WebKitTestRunner/InjectedBundle/cocoa/AccessibilityCommonCocoa.mm:
* Tools/WebKitTestRunner/InjectedBundle/ios/AccessibilityControllerIOS.mm:
* Tools/WebKitTestRunner/InjectedBundle/ios/AccessibilityTextMarkerIOS.mm:
* Tools/WebKitTestRunner/InjectedBundle/mac/AccessibilityNotificationHandler.mm:
* Tools/WebKitTestRunner/InjectedBundle/mac/AccessibilityUIElementMac.mm:
(WTR::AccessibilityUIElement::boundsForRange):
* Tools/WebKitTestRunner/InjectedBundle/win/AccessibilityUIElementWin.cpp:

Canonical link: https://commits.webkit.org/298630@main
